core/benches/arrays.rs

```diff
@@ -1,6 +1,7 @@
 use std::rc::Rc;
 
 use criterion::{criterion_main, Criterion};
+use nickel_lang_core::cache::InputFormat;
```

Comment:
(See comment in `ncl_bench_group`, but example of import that we shouldn't have to do)